Src/WitsmlExplorer.Frontend/components/Modals/ServerModal.tsx

```diff
@@ -133,76 +196,186 @@ const ServerModal = (props: ServerModalProps): React.ReactElement => {
+      setPassword(serverCredentials.password);
+    }
+  }, [serverCredentials]);
+
+  const isCompactMode = useTheme().props.MuiCheckbox.size === "small";
+  const setStandAlonePopup: ReactElement = 
+    <React.Fragment key={"standalone"}>
+      {
+        standAlone ? <div style={standAlone?Styles.noServerText:<></>}>
+          <Typography style={Styles.noServer} token={{
+            color: colors.interactive.primaryResting,
+            fontWeight: 500,
+          }} >
+            <Icons name='new_alert' size={32} />
+            No server Connected!
+          </Typography>
+          {standAlone && <Typography color="danger"  token={{fontWeight: 500,fontSize: '1rem',}} style={Styles.errorText} >{errorMessage}</Typography>}
+        </div> : ""
+      }
+      <Section style={standAlone?Styles.noServerContent:Styles.feildname} compactMode={isCompactMode}>
+        <Label label="Server URL" style={ Styles.feildname} />
+        <TextField
+          id="url"
+          className="fieldInputs"
+          size="small"
+          variant="outlined"
+          defaultValue={server?.url}
+          error={displayUrlError}
+          helperText={displayUrlError ? "Not a valid server url" : ""}
+          fullWidth
+          inputProps={{ maxLength: 256 }}
+          onChange={onChangeUrl}
+          onBlur={runUrlValidation}
+          required />
+        <Label label="Server Name" style={Styles.feildname} />
+        <TextField
+          id="name"
+          className="fieldInputs"
+          size="small"
+          variant="outlined"
+          defaultValue={server?.name}
+          error={displayNameError}
+          helperText={displayNameError ? "A server name must have 1-64 characters" : ""}
+          fullWidth
+          inputProps={{ minLength: 1, maxLength: 64 }}
+          onBlur={runServerNameValidation}
+          onChange={onChangeName}
+          required />
+        <Label label="Username" style={Styles.feildname} />
+        <TextField
+          id="username"
+          className="fieldInputs"
+          size="small"
+          variant="outlined"
+          helperText={username?.length === 0 ? "Not a valid UserName" : ""}
+          defaultValue={username}
+          fullWidth
+          onChange={(e) => setUsername(e.target.value)}
+          inputProps={{ minLength: 1, maxLength: 64 }}
+          required />
+        <Label label="Password" style={Styles.feildname} />
+        <TextField
+          id={"password"}
+          variant="outlined"
+          className="fieldInputs"
+          size="small"
+          defaultValue={password}
+          helperText={password?.length === 0 ? "Password must be 1-7936 characters" : ""}
+          fullWidth
+          type="password"
+          autoComplete="current-password"
+          inputProps={{ minLength: 1, maxLength: 7936 }}
+          onChange={(e) => setPassword(e.target.value)}
+          required />
+        <Label label="Version" style={Styles.feildname} />
```

Comment:
This is probably ment to be the `SecurityScheme` or `Scheme` instead of `Version`

The field should not be clearable and can be disabled like in this PR: https://github.com/equinor/witsml-explorer/pull/1657/files

There will also be need for a field for app `roles` assigned to the server (configured in Azure) like below. 

![image](https://user-images.githubusercontent.com/1442452/204513171-8750600d-2fe3-4648-8d36-189e83ba1bc7.png)